packages/wallet-connectors/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@concordium/wallet-connectors",
-    "version": "0.3.2",
+    "version": "0.4.0-rc.6",
```

Comment:
same, also below

Reply:
Yes, published beta version for testing under beta tag on NPM